modules/header/react/lib/parts/Search.tsx

```diff
@@ -5,11 +5,10 @@ import {CSSTransition} from 'react-transition-group';
 import {HeaderHeight, HeaderTheme} from '../shared/types';
 import {colors, spacing, spacingNumbers, type} from '@workday/canvas-kit-react-core';
 import {focusRing} from '@workday/canvas-kit-react-common';
-import {SystemIcon} from '@workday/canvas-kit-react-icon';
 import {IconButton} from '@workday/canvas-kit-react-button';
 import {searchIcon, xIcon, xSmallIcon} from '@workday/canvas-system-icons-web';
 
-export type SearchProps = {
+export interface SearchProps extends React.HtmlHTMLAttributes<HTMLFormElement> {
```

Comment:
I didn't see anything that required this change. Is this change to be consistent with `HeaderProps`? Do we have documentation around prop passing as an API convention? What about child element attributes?

Reply:
https://github.com/Workday/canvas-kit/blob/master/API_PATTERN_GUIDELINES.md#prop-spread-behavior